src/shared/util.js

```diff
@@ -1219,16 +1219,111 @@ function MessageHandler(sourceName, targetName, comObj) {
   this.targetName = targetName;
   this.comObj = comObj;
   this.callbackIndex = 1;
+  this.streamId = 1;
```

Comment:
> Yes, sure. I will change it to `streamIndex`

Thanks, but I now notice that this may become problematic with respect to the line length limit; it may be simpler to keep `this.streamId` and just change `this.callbackIndex` -> `this.callbackId` instead :-)

Looking at the existing code, the `callbackId` name is actually already used in most places, except here: https://github.com/mozilla/pdf.js/blob/master/src/shared/util.js#L1313.